tests/integration/test_tf_integration.py

```diff
@@ -422,3 +422,25 @@ def test_coherent_dm_gradient(self, setup_eng, cutoff, tol, batch_size):
         grad = tape.gradient(res, [a])
         grad_ex = -a ** 3 * (a ** 2 - 2) * np.exp(-a ** 2)
         assert np.allclose(grad, grad_ex, atol=tol, rtol=0)
+
+
+def test_TODO():
```

Comment:
I'm curious about this test name :thinking:

Reply:
No docstring either

Reply:
Yes this shouldn't be here (yet), I think it was a bit of a miscommunication on the PR being ready to merge.

The idea of this test is to compare gradients using old and new versions of einsum. The plan is to assert that the gradients should be the same and add an xfail to show that we expect failure - when this stops being the case we know they've fixed the bug.

Unfortunately I couldn't get it to work and was going to come back to this today, hence the TODO and the print statements - it's just in an early prototype form.

Reply:
Since this is merged into the `tf2` branch and has not yet made it to `master`, we can sort this out by doing a follow up PR to properly add the test.

Reply:
Oh wait, @josh146 already fixed it! https://github.com/XanaduAI/strawberryfields/commit/932856b9e399331be7966c7d962964f8d750f361 :100: